code/script.js

```diff
@@ -1,47 +1,177 @@
+   showMessage(`Yummy in my tummy! I also love ${type} What kind of topping would you like?`, 'bot') 
+  
 
-// When website loaded, chatbot asks first question.
-// normally we would invoke a function like this:
-// greeting()
-// But if we want to add a little delay to it, we can wrap it in a setTimeout:
-// setTimeout(functionName, timeToWaitInMilliSeconds)
+    if (type === 'Princess Cake') {
+      form.innerHTML = `
+      <select id="select" class="topping"> 
+      <option value="" selected disabled>Pick your topping!</option>
+      <option value="Icing Sugar">Icing sugar</option>
+      <option value="Strawberries">Strawberries</option>
+      <option value="Marzipan Pigs">Marzipan Pigs</option>
+      </select> 
+      `
+    }
+    else if (type === 'Pancake Cake') {
+      form.innerHTML = `
+        <select id="select" class= topping>
+          <option value="" selected disabled>Pick your topping!</option>
+          <option value="Chocolate Sauce">Chocolate Sauce</option>
+          <option value="Sprinkles">Sprinkles</option>
+          <option value="Whipped Cream">Whipped Cream</option>
+        </select>
+      `
+    } else {
+      form.innerHTML = `
+        <select id="select" class="topping">
+          <option value="" selected disabled>Pick your topping!</option>
+          <option value="Blue Berries">Blue Berries</option>
+          <option value="Chocolate Sprinkles">Chocolate Sprinkles</option>
+          <option value="Banana">Banana</option>
+        </select>
+      `
+    }
+    const select = document.getElementById('select')
+    select.addEventListener('change', () => cakePieces(select.value))
+     
+    // select.value ni skickar med det till nästa funktion men använder det inte där consol logga det.
+  }
+    const cakePieces = (topping) => {
+    cakeTopping = topping
+    showMessage(cakeTopping, 'user')
+    showMessage(`${cakeTopping} is a great choice of topping! How many pieces would you like? ${names}`, 'bot')
+    form.innerHTML = `
+    <button id="fourBtn">Four</button>
+    <button id="sixBtn">Six</button>
+    <button id="twelveBtn">Twelve</button>
+  `
+
+  document
+    .getElementById('fourBtn')
+    .addEventListener('click', () => delivery('four', 100))
+  document
+    .getElementById('sixBtn')
+    .addEventListener('click', () => delivery('six', 150))
+  document
+    .getElementById('twelveBtn')
+    .addEventListener('click', () => delivery('twelve', 200))
+  }
+
+ const delivery = (pieces, price) => {
+   thePieces = pieces
+  thePrice = price
+  showMessage(pieces, 'user')
+  showMessage(`When do you want your cake delivered?`, 'bot')
+   form.innerHTML = `
+   <label for="start" id="dateForm"></label>
+
+  <input type="date" id="dateInput" name="trip-start">
+  <button id="sendDate">send</button>
+   `
+   document.getElementById("dateInput").addEventListener("change", function() {
+     const input = this.value;
+     var dateEntered = new Date(input);
+     theDate = input
```

Comment:
Does this store the value from the calendar? Would have been awesome to include on the order confirmation!